working/augmentation-libraries/feature-specification.md

```diff
@@ -122,30 +122,34 @@ syntax][import]?**
 
 [import]: https://github.com/dart-lang/language/blob/master/working/0649%20-%20Import%20shorthand/proposal.md
 
-The URL points to the main library that this augmentation is applied to.
+The URL points to the augmented library that this augmentation is applied to,
+which must be a regular library (and not a library augmentation).
```

Comment:
The parenthetical implies that a hand-authored library augmentation can't contain macro applications. Is that deliberate?

The "Merge Order" section still says that an entire tree of library augmentations is possible. Or is it that every library augmentation must point all the way back to the augmented library and not any of the library augmentations?